src/System.ComponentModel.TypeConverter/ref/System.ComponentModel.TypeConverter.cs

```diff
@@ -8,6 +8,7 @@
 using System.Runtime.Serialization;
 
 [assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.Component))]
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.ComponentModel.ISupportInitialize))]
```

Comment:
We need this for System.Data.Common tests if I don't add the typeforward they wouldn't compile. Is there a better solution for that?